src/utils/api/transaction/lsk.js

```diff
@@ -304,28 +304,31 @@ export const getTransactionFee = async ({
  */
 export const create = ({
   network,
-  moduleAssetId,
-  keys,
-  ...transactionObject
+  account,
+  passphrase,
+  transactionObject,
 // eslint-disable-next-line max-statements
 }) => new Promise((resolve, reject) => {
   const { networkIdentifier } = network.networks.LSK;
   const {
-    passphrase, ...rawTransaction
+    moduleAssetId, ...rawTransaction
   } = transactionObject;
 
+  rawTransaction.nonce = account.sequence.nonce;
+  rawTransaction.senderPublicKey = account.summary.publicKey;
+
   const schema = moduleAssetSchemas[moduleAssetId];
   const transaction = createTransactionObject(rawTransaction, moduleAssetId);
   const netId = Buffer.from(networkIdentifier, 'hex');
 
   try {
     let signedTransaction;
 
-    if (keys.numberOfSignatures > 0
+    if (account.keys.numberOfSignatures > 0
```

Comment:
I think the better check is `account.summary.isMultiSignature`

Reply:
Done.